src/pages/ProductDetailPage.tsx

```diff
@@ -0,0 +1,117 @@
+import Spacing from "@/components/Spacing";
+import styled from "@emotion/styled";
+import { useNavigate, useParams } from "react-router-dom";
+import { Suspense } from "react";
+import ProductCard from "@/components/productDetail/ProductCard";
+import DetailCard from "@/components/productDetail/DetailCard";
+import WishButton from "@/components/productDetail/WishButton";
+import { ErrorBoundary } from "@/components/ErrorBoundary";
+import LoadingSpinner from "@/components/LoadingSpinner";
+import { getUserFromSession } from "@/utils/getUserFromStorage";
+import { PATH } from "@/paths";
+
+function ProductDetailContent() {
+  const { productId } = useParams<{ productId: string }>();
```

Comment:
아래코드에서 productId가 없을 때 빈 문자열을 넘겨주는 코드가 반복되는데요!
기본값을 할당해주면 중복 제거가 가능해보여요~
```suggestion
  const { productId = '' } = useParams<{ productId: string }>();
```